### ISSUE TITLE
Microsoft Edge has wrong id on charts

### ISSUE THREAD

**Issue by Orange-kun** [WebPlatformTest/HTML5test#567]
Microsoft Edge still shows as current version 17 and upcoming version 18 both with outdated scores. Now Canary Edge version is 84.0.516 and has 532 score; please fix yor diagrams and remove legacy edge replacing it with Chromium Edge

**Comment by prirai** [comment]
Yes, I too found it today. I have edge chromium 86 and it shows my score as 528.
